### DISCUSSION TITLE
A Problem about Explorer's Search Bar

### DISCUSSION THREAD

**Discussion by Listerdale** [valinet/ExplorerPatcher#572] [Q&A]
As the picture.
![图片](https://user-images.githubusercontent.com/96236162/146393546-73dc2498-e56b-46a6-9a2e-1d5e7b747c00.png)
The magnifying glass icon will show as a character if it isn't selected.
![图片](https://user-images.githubusercontent.com/96236162/146393831-8608c104-0d58-45a3-ab70-c8d69e4d5c16.png)
It will be normal while selecting it.
Can anyone solve the problem? Thanks!

**Comment by valinet** [comment]
That's likely some bug or quirk in Windows. I don't experience that, granted my system is set to US English.
